src/screens/send/styles.ts

```diff
@@ -25,7 +25,7 @@ export default (theme: ITheme) =>
         inputAddress: {
             flex: 1,
             color: theme.colors.text,
-            paddingRight: 40
+            paddingRight: 15
```

Comment:
use the new const @maricalucian added, i think there is no problem if padding would be 16